dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/WebSocketPrivateStreamTestSuite.scala

```diff
@@ -150,18 +144,18 @@ class WebSocketPrivateStreamTestSuite extends MatcherSuiteBase with HasWebSocket
       wsc.getAllOrders.distinct should matchTo(
         Seq(
           WsOrder.fromDomain(LimitOrder(bo1), OrderStatus.Accepted),
-          WsOrder.orderDiff(LimitOrder(bo1), OrderStatus.Filled.name.some, 10.0, 0.003, 1.0)
+          WsOrder.fromDomain(LimitOrder(bo1), status = OrderStatus.Filled(0.waves, 0.waves), 10.0, 0.003, 1.0)
```

Comment:
Here I meant that `status` here should be equal to `OrderStatus.Filled(10.waves, 0.003.waves)`
Actually it doesn't matter what you put here, status is only used for it's name deriving

Reply:
ok